src/pilot/install-director.sh

```diff
@@ -261,8 +268,16 @@ apply_patch "sudo patch -b -s /usr/lib/python2.7/site-packages/dracclient/wsman.
 sudo rm -f /usr/lib/python2.7/site-packages/dracclient/wsman.pyc
 sudo rm -f /usr/lib/python2.7/site-packages/dracclient/wsman.pyo
 
+# This hacks in a patch to enable realtime RAID creation.
+echo
+echo "## Patching Ironic iDRAC driver job.py..."
+apply_patch "sudo patch -b -s /usr/lib/python2.7/site-packages/dracclient/resources/job.py ${HOME}/pilot/job.patch"
+sudo rm -f /usr/lib/python2.7/site-packages/dracclient/job.pyc
+sudo rm -f /usr/lib/python2.7/site-packages/dracclient/job.pyo
```

Comment:
I believe the path to the files being removed by the two (2) lines above is incorrect. Shouldn't the path be `/usr/lib/python2.7/site-packages/dracclient/resources/` ?